notifikasjon/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/river/OpprettForespoerselSakOgOppgaveRiverTest.kt

```diff
@@ -0,0 +1,201 @@
+                    harddeleteOm = sakLevetid,
+                )
+                mockAgNotifikasjonKlient.opprettNyOppgave(
+                    virksomhetsnummer = innkommendeMelding.orgnr.verdi,
+                    eksternId = innkommendeMelding.forespoerselId.toString(),
+                    grupperingsid = innkommendeMelding.forespoerselId.toString(),
+                    merkelapp = NotifikasjonTekst.MERKELAPP,
+                    lenke = "en-slags-url/im-dialog/${innkommendeMelding.forespoerselId}",
+                    tekst = NotifikasjonTekst.OPPGAVE_TEKST,
+                    varslingTittel = NotifikasjonTekst.STATUS_TEKST_UNDER_BEHANDLING,
+                    varslingInnhold = NotifikasjonTekst.oppgaveInnhold(innkommendeMelding.orgnr, innkommendeMelding.orgNavn),
+                    tidspunkt = null,
+                )
+            }
+        }
+
+        test("ukjent feil for sak håndteres") {
+            val innkommendeMelding = innkommendeMelding()
+
+            coEvery {
+                mockAgNotifikasjonKlient.opprettNySak(any(), any(), any(), any(), any(), any(), any(), any())
+            } throws NullPointerException("To me, religion is like Paul Rudd.")
+
+            testRapid.sendJson(innkommendeMelding.toMap())
+
+            testRapid.inspektør.size shouldBeExactly 1
+
+            testRapid.firstMessage().toMap() shouldContainExactly forventetFail(innkommendeMelding).tilMelding()
+
+            coVerifySequence {
+                mockAgNotifikasjonKlient.opprettNySak(any(), any(), any(), any(), any(), any(), any(), any())
+            }
+        }
+
+        test("ukjent feil for oppgave håndteres") {
+            val innkommendeMelding = innkommendeMelding()
+
+            coEvery {
+                mockAgNotifikasjonKlient.opprettNySak(any(), any(), any(), any(), any(), any(), any(), any())
+            } returns MOCK_SAK_ID
+
+            coEvery {
+                mockAgNotifikasjonKlient.opprettNyOppgave(any(), any(), any(), any(), any(), any(), any(), any(), any())
+            } throws NullPointerException("Doing anything more than the minimum amount of work required is my definition of failing.")
+
+            testRapid.sendJson(innkommendeMelding.toMap())
+
+            testRapid.inspektør.size shouldBeExactly 1
+
+            testRapid.firstMessage().toMap() shouldContainExactly forventetFail(innkommendeMelding).tilMelding()
+
+            coVerifySequence {
+                mockAgNotifikasjonKlient.opprettNySak(any(), any(), any(), any(), any(), any(), any(), any())
+                mockAgNotifikasjonKlient.opprettNyOppgave(any(), any(), any(), any(), any(), any(), any(), any(), any())
+            }
+        }
+
+        context("ignorerer melding") {
+            withData(
+                mapOf(
+                    "melding med behov" to Pair(Key.BEHOV, BehovType.LAGRE_SELVBESTEMT_IM.toJson()),
+                    "melding med data" to Pair(Key.DATA, "".toJson()),
```

Comment:
Dette gjelder bare for meldinger med data med tom verdi antar jeg?
Hva med "melding med data med tom verdi" feks?

Reply:
Jess, skal fikse. Det ble kopiert fra en river som avviser `Key.DATA` blankt.